src/components/FormCheckbox.vue

```diff
@@ -73,7 +93,7 @@ export default {
         },
         trueValue: {
             type: [String, Number, Boolean],
```

Comment:
keď to má byť html input, tak to neho nevieš zapísať boolean ani číslo.. input value môže byť len string.. samozrejme že s v-model pôjde všetko.. stále ale by sme sa mali snažiť byť čo najviac natívny.. mať <form><input name=blabla value=1 type=checkbox></form> a posielať dáta cez new FormData(formElement), žiadne buildenie dát s JS